pkg/sigretests/main.go

```diff
@@ -48,6 +50,20 @@ func finishedJSONURL(org string, repo string, prNumber string, jobName string, b
 	return fmt.Sprintf("%s/%s/%s/finished.json", prStorageURL(org, repo, prNumber), jobName, buildNumber)
 }
 
+func checkSIGCIFailure(job job) bool {
+	if job.failure {
+		junitURL := fmt.Sprintf("%s/junit.functest.xml", job.artifactsURL)
+		resp, err := http.Get(junitURL)
+		if resp.StatusCode != http.StatusOK {
+			return true
+		}
+		if err != nil {
```

Comment:
I'd swap the order - rather check error code first. Or is there a reason for this order?

Reply:
yes sure can swap the order